pkg/datastore/postgres/repositories/checked_headers_repository_test.go

```diff
@@ -119,6 +119,38 @@ var _ = Describe("Checked Headers repository", func() {
 		})
 	})
 
+	Describe("MarkHeadersUnchecked", func() {
+		It("removes rows for headers <= starting block number", func() {
```

Comment:
maybe more accurate to say "marks header with matching block number unchecked" or something?